### PR TITLE
add note about browser extensions to 3P passkey provider definition

### LINKED ISSUE #280: Passkey providers via a browser extension
# Existing content does not mention browser extensions as a way of providing passkey support

## Link to content
https://github.com/passkeydeveloper/passkeys.dev/blob/464d98d952fe066a4d890843d540ef74f1d43806/content/en/docs/reference/terms/index.md?plain=1#L167

### What is the issue?

Many 3rd party passkey providers, password managers, provide support for passkeys via a browser extension. The audience reading content on this site, who may be implementing passkeys need to be aware of this and test against it during development. As an example, decisions may be made based on user agent info (version of macOS etc) to create UX that tells their users whether or not a passkey can be saved to their password manager, we've seen this already in the wild.

This is a reality their users will face so it should be accounted for in some manner.

### Proposed changes

Browser extensions that handle passkeys should be mentioned in the documentation in some way. Even if it is to say something along the lines of "some passkey providers provide support via a browser extension, which can be considered a temporary solution while a more robust solution is being developed."

### Any other notes

Password managers that support passkeys via a browser extension:

- 1Password
- LastPass
- Bitwarden (planned)
- Keeper Security
- NordPass

### PR DESCRIPTION
**Pull request by timcappalli**
Resolves #280

### CODE REVIEW

File: content/en/docs/reference/terms/index.md [hunk 1]
```diff
@@ -172,6 +172,8 @@ A FIDO2 [Discoverable Credential](#discoverable-credential) that can reliably be
 
 A [Passkey Provider](#passkey-provider) that plugs in to the OS via platform APIs to store and manage a user's passkeys via the platform authenticator.
 
+> NOTE: Some passkey providers intercept WebAuthn requests using browser extensions which may bypass browser and/or platform interfaces and features.
```
Comment (irew) [inline content/en/docs/reference/terms/index.md:175]:
```suggestion
> NOTE: Some passkey providers intercept WebAuthn requests in order to provide passkey support using browser extensions which may bypass browser and/or platform interfaces and features.
```

**Review by irew — APPROVED** [review]
lgtm

File: content/en/docs/reference/terms/index.md [hunk 2]
```diff
@@ -172,6 +172,8 @@ A FIDO2 [Discoverable Credential](#discoverable-credential) that can reliably be
 
 A [Passkey Provider](#passkey-provider) that plugs in to the OS via platform APIs to store and manage a user's passkeys via the platform authenticator.
 
+> NOTE: Some passkey providers intercept WebAuthn requests in order to provide support for passkeys using browser extensions which may bypass browser and/or platform interfaces and features.
```
Comment (MasterKale) [inline content/en/docs/reference/terms/index.md:175]: Suggesting some alternate wording:

```suggestion
> NOTE: Some passkey providers support passkeys via a browser extension that intercepts WebAuthn requests. This bypasses browser and/or platform interfaces and features.
```

Comment (timcappalli) [reply content/en/docs/reference/terms/index.md:175]: fine with me. @irew?

Comment (irew) [reply content/en/docs/reference/terms/index.md:175]: While it does bypass, it doesn't mean the user is unable to access the browser/platform experience. That nuance should be captured. As an example, Dashlane will hand back to the browser/platform if the authenticator attachment is "cross-platform". Plus users are always able to fallback to the browser/platform if they need to.

Comment (MasterKale) [reply content/en/docs/reference/terms/index.md:175]: @irew What about something like this?

```suggestion
> NOTE: Some passkey providers support passkeys via a browser extension that intercepts WebAuthn requests. Providers that bypass browser and/or platform interfaces and features in this manner typically offer a way for the intercepted request to be passed back to the browser and/or platform to handle as usual.
```

Comment (irew) [reply content/en/docs/reference/terms/index.md:175]: Lgtm 👍🏽

**Review by MasterKale — APPROVED** [review]

**Pull request merged**